tests/README.md

```diff
@@ -1,6 +1,40 @@
 # `stackstorm-ha` Helm chart Tests
 
-This directory contains Helm chart tests, powered by [BATS](https://github.com/sstephenson/bats) Bash Automated Testing System.
+This directory contains Helm chart integration (under `tests/integration/`) and unit tests (`tests/unit-*_test.yaml`).
+
+## Unit tests
+
+Unit tests (`tests/unit-*_test.yaml`) use [`helm-unittest`](https://github.com/quintush/helm-unittest).
+`helm-unittest` uses a yaml-based test file to ensure that the templates generate expected features.
+For example, they can ensure that custom annotations are applied consistently to all of the deployments.
+Unit tests do not require a running kubernetes cluster.
+
+Before running unit tests, install the `helm-unittest` plugin and ensure you have sub-charts installed:
+```
+helm plugin install https://github.com/quintush/helm-unittest
+helm dependency update
+```
+
+To run the tests manually from the chart's root dir:
+```
+helm unittest --helm3 .
+```
+
+Note that `helm-unittest` still defaults to helm 2, so you must pass `--helm3` or `-3` for short.
+`helm-unittest` does not have a configuraiton file which makes using non-standard directories (like `tests/unit/`)
+very inconvenient (you would have to specify `-f tests/unit/*_test.yaml` on every invocation).
+So, we keep all of the unit test files under tests, but prefix them with `unit-` to separate them from
+the integration tests.
```

Comment:
```suggestion
```
Remove. It may sound a bit like a complaint, rather than documentation, which is not what we need in the docs 🙂 Just providing a CLI example was sufficient.